provisioning.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
```

Comment:
This file isn't actively used anywhere and should be deprecated by #422 in the future, but since it's still in the repo, I've updated it.